Android/MMKV/mmkv/src/main/cpp/MMKV.cpp

```diff
@@ -1213,14 +1217,27 @@ static void mkSpecialCharacterFileDirectory() {
     free(path);
 }
 
+static void md5(string &value) {
+    unsigned char md[MD5_DIGEST_LENGTH];
+    char tmp[3] = {'\0'}, buf[33] = {'\0'};
+    MD5((const unsigned char *)value.c_str(), value.size(), md);
+    for(int i = 0;i < MD5_DIGEST_LENGTH;i++)
+    {
+        sprintf(tmp, "%2.2x", md[i]);
+        strcat(buf, tmp);
+    }
+    value = buf;
+}
+
 static string encodeFilePath(const string &mmapID) {
 	const char *specialCharacters = "\\/:*?\"<>|";
 	string filePath(mmapID);
```

Comment:
This string copy in unnecessary. I suggest making `void md5(string&)` into `string md5(const string&)`.